thoth/storages/graph/postgres.py

```diff
@@ -3675,26 +3676,71 @@ def get_active_kebechet_github_installations_repos_count_all(self) -> int:
 
     def get_kebechet_github_installations_active_managers(self, slug: str) -> list:
         """Return the list of active managers for a particular repository.
-        Passed a slug name to be deactivated.
-        Example - slug:'thoth-station/advisor'
-        :rtype: List
-        :returns List of manager currently activated for the repo.
+
+        Examples:
+        >>> from thoth.storages import GraphDatabase
+        >>> graph = GraphDatabase()
+        >>> graph.get_kebechet_github_installations_active_managers(
+            slug="thoth-station/srcops-testing"
+        )
+        ['thoth_advise_manager']
         """
         with self._session_scope() as session:
             instance = (
                 session.query(KebechetGithubAppInstallations)
                 .filter(KebechetGithubAppInstallations.is_active.is_(True))
                 .filter(KebechetGithubAppInstallations.slug == slug)
+                .with_entities(
+                    KebechetGithubAppInstallations.info_manager,
+                    KebechetGithubAppInstallations.pipfile_requirements_manager,
+                    KebechetGithubAppInstallations.update_manager,
+                    KebechetGithubAppInstallations.version_manager,
+                    KebechetGithubAppInstallations.thoth_advise_manager,
+                    KebechetGithubAppInstallations.thoth_provenance_manager,
+                )
                 .first()
             )
-            if instance:
-                active_managers = []
-                for attr in dir(instance):
-                    if attr.endswith("manager") and not callable(getattr(instance, attr)):
-                        if getattr(instance, attr) == True:
-                            active_managers.append(attr)
-                return active_managers
-            return []
+
+            return [manager for manager, is_active in instance._asdict().items() if is_active == True]
+
+    def get_kebechet_github_installations_active_managers_count_all(
+        self,
+        kebechet_manager: str,
+    ) -> int:
+        """Return the number of repos with specific manager active.
+
+        :rtype: int
+        :returns Number of active records for registered Kebechet manager.
+        """
+        if kebechet_manager not in KebechetManagerEnum._member_names_:
```

Comment:
where is the `._member_names_` this attribute coming from ?

Reply:
it is a built in attribute from Enum class, which is base class for KebechetManagerEnum and all Enums we use :)